runtime/memory/cortex_m_wrapping.c

```diff
@@ -27,6 +27,19 @@ void expand_memory() {
     memory_size += WASM_PAGE_SIZE;
 }
 
+i32 instruction_memory_size() {
+    return memory_size / WASM_PAGE_SIZE;
+}
+
+i32 instruction_memory_grow(i32 count) {
```

Comment:
This feels pretty generic, but is replicated across backends. I wonder if there's an opportunity for more code sharing. Perhaps a future issue?

Reply:
I think we can address in this PR. Maybe @geky can lift these to `runtime.c`?

Reply:
Assuming expand_memory, memory_size, and WASM_PAGE_SIZE are all available globally I can move these to runtime.c.

Though I wonder if memory_grow/memory_size would be a better interface than expand_memory/memory_size. They are isomorphic to expand_memory/memory_size, but are closer the Wasm specification.